packages/flutter_tools/templates/plugin_ffi/src.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,23 @@
+# We use the CMake version connected to the version of the Android Gradle Plugin.
+#
+# CMake 3.10.2.
+# https://developer.android.com/studio/projects/install-ndk#vanilla_cmake
+#
+# Changing the CMake version requires all users of this plugin to
```

Comment:
"clients" (or "customers")

Reply:
It's actually worse than that; at least on Windows they would need to either:
- download a newer version of Visual Studio, or
- do surgery on the internals of their VS installation.

I would phrase this more strongly, as in the Windows and Linux plugin comments I added:
"Changing the CMake version will cause the plugin to fail to compile for some
customers of the plugin."

Reply:
I've replaced the message with the same one you added on the other CMakeLists files:

```
# The Flutter tooling requires that developers have CMake 3.10 or later
# installed. You should not increase this version, as doing so will cause
# the plugin to fail to compile for some customers of the plugin.
```